core/src/main/java/hudson/slaves/ChannelPinger.java

```diff
@@ -136,6 +150,7 @@ public void onClosed(Channel channel, IOException cause) {
         });
 
         t.start();
-        LOGGER.fine("Ping thread started for " + channel + " with a " + interval + " minute interval");
+        LOGGER.fine("Ping thread started for " + channel + " with a " +
+            intervalSeconds + " second interval and a " + timeoutSeconds + " second timeout.");
```

Comment:
Logger.log(Level.FINE, "This is a {0} {1} {2}", "paramaterised", "message", "in a log");

Using this the string concatenation will not take place if the log is not enabled (whereas the code in ln187 will always perform String concatenation).